tests/client/main.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define CATCH_CONFIG_MAIN
+
+#include "api_internal.h"
+#include "catch2\catch.hpp"
+#include "ebpf_api.h"
+#pragma warning(push)
+#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
+                                // 'type2', possible loss of data
+#undef VOID
+#include "ebpf_verifier.hpp"
+#pragma warning(pop)
+#include "header.h"
+#include "rpc_client.h"
+#include "rpc_interface_h.h"
+#include "service_helper.h"
+
+#define EBPF_SERVICE_BINARY_NAME L"ebpfsvc.exe"
+#define EBPF_SERVICE_NAME L"ebpfsvc"
+
+service_install_helper service_helper(EBPF_SERVICE_NAME, EBPF_SERVICE_BINARY_NAME);
+
+int
+ebpf_rpc_verify_program(ebpf_program_verify_info* info, char** logs, uint32_t* logs_size);
```

Comment:
Assuming these are defined in another file, move these two prototypes to a .h file.

Reply:
ebpf_rpc_verify_program to header file. service_helper is a global object.